project/Dependencies.scala

```diff
@@ -114,7 +115,7 @@ object Dependencies {
 
   val tensorflow = l ++= tensorflowDeps ++ Seq(Test.scalaTest)
 
-  val xgboostRuntime = l ++= Seq(xgboostDep) ++ Seq(Test.scalaTest)
+  val xgboostRuntime = l ++= Seq(xgboostDep) ++ Test.spark ++ Seq(Test.scalaTest)
```

Comment:
are you sure Provided.spark isn't enough? it seems like that for the other parity tests?

Reply:
So I actually meant to get your input on these dependency changes. 
Both Provided.spark or Test.spark achieve the same thing here because I am using spark things only in tests.

Now, I think that putting those dependencies with a `test` <scope> makes the build faster, since the dependencies are only brought in when running tests, not when building. I read online:

- test: 
> Dependencies with maven dependency scope test are not needed to build and run the project. They are needed to compile and run the unit tests.

- provided:
> Maven dependency scope provided is used during build and test the project. 

so I assume a test scope is slightly lighter than provided? Again, the xgboost-runtime code has no dependencies on spark stuff, but its tests do

Reply:
That makes sense, thank you!